### PR TITLE
added an exercise for dynamic highline simulation

### LINKED ISSUE #56: Hängebrückenaufgabe aus SoSe 2017
Leicht abgeändert kann man die ins Eigenwerte-Kapitel mitaufnehmen

### PR DESCRIPTION
**Pull request by PhiSpel**
included the P4 exercise from #56 except exercise 2 (which was the static case adressed in [04_lineare_gleichungen](https://github.com/joergbrech/Modellbildung-und-Simulation/tree/master/content/04_lineare_gleichungen)

should close #56 unless additional input is wanted

simulating as a PDE is mentioned in #123

### CODE REVIEW

File: content/07_num_ode/03_modellierung_einer_highline_dynamsich.ipynb [hunk 1]
```diff
@@ -0,0 +1,423 @@
+{
```
Comment (sandro-elsweijer) [inline content/07_num_ode/03_modellierung_einer_highline_dynamsich.ipynb:1]: Typo in file name

File: content/07_num_ode/03_modellierung_einer_highline_dynamsich.ipynb [hunk 2]
```diff
@@ -0,0 +1,423 @@
...
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "30c0462e",
+   "metadata": {},
+   "source": [
+    "# Modellierung einer Highline - Zeitliche Auflösung\n",
+    "\n",
+    "Sie kennen dieses Modell einer Highline bereits aus dem Kapitel zu linearen Gleichungen.\n",
```
Comment (sandro-elsweijer) [inline content/07_num_ode/03_modellierung_einer_highline_dynamsich.ipynb:10]: We could reference the other chapter here (https://jupyterbook.org/content/references.html#references-and-cross-references)

File: content/07_num_ode/03_modellierung_einer_highline_dynamsich.ipynb [hunk 3]
```diff
@@ -0,0 +1,423 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f0a3b7de",
+   "metadata": {},
+   "source": [
+    "## Aufgabe 1\n",
+    "\n",
+    "Lösen Sie die Differentialgleichung mit der Matlab–Routine `ode45` auf einem Zeitinterval von 90 Sekunden. Als externte Kraft Fext soll zunächst nur die Schwerkraft auf die Brückenknoten wirken. Verwenden Sie $\\mathbf{z}(0)=\\mathbf{\\dot{z}}(0)=0$ als Anfangsbedingungen. Zunächst nehmen wir $\\mathbf{F}_{ext} = \\text{const.} = \\mathbf{F}_g$ an.\n",
```
Comment (sandro-elsweijer) [inline content/07_num_ode/03_modellierung_einer_highline_dynamsich.ipynb:82]: Format of F_ext not in math mode

File: content/07_num_ode/03_modellierung_einer_highline_dynamsich.ipynb [hunk 4]
```diff
@@ -0,0 +1,423 @@
...
+   "metadata": {
+    "tags": [
+     "hide_input"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "classdef wind < handle\n",
+    "    % pedestrians is an object to create the excitation caused by\n",
+    "    % pedestrians on a hanging bridge.\n",
```
Comment (sandro-elsweijer) [inline content/07_num_ode/03_modellierung_einer_highline_dynamsich.ipynb:313]: We should adapt the description for the wind class

File: content/07_num_ode/wind.m [hunk 5]
```diff
@@ -0,0 +1,21 @@
+classdef wind < handle
+    % pedestrians is an object to create the excitation caused by
+    % pedestrians on a hanging bridge.
```
Comment (sandro-elsweijer) [inline content/07_num_ode/wind.m:3]: s. o.

**Review by sandro-elsweijer — CHANGES_REQUESTED** [review]
I didn't check if the code works, but I think you checked it with your solution.
We should open the mentioned issue before merging, so we do not forget it afterwards.

File: content/07_num_ode/03_modellierung_einer_highline_dynamsich.ipynb (hunk 1 above)
Comment (PhiSpel) [reply content/07_num_ode/03_modellierung_einer_highline_dynamsich.ipynb]: solved in e9b9f8f

File: content/07_num_ode/03_modellierung_einer_highline_dynamsich.ipynb (hunk 2 above)
Comment (PhiSpel) [reply content/07_num_ode/03_modellierung_einer_highline_dynamsich.ipynb]: great reference, thank you!

solved by b49d510

File: content/07_num_ode/03_modellierung_einer_highline_dynamsich.ipynb (hunk 3 above)
Comment (PhiSpel) [reply content/07_num_ode/03_modellierung_einer_highline_dynamsich.ipynb]: this was not the case for me..

Comment (sandro-elsweijer) [reply content/07_num_ode/03_modellierung_einer_highline_dynamsich.ipynb]: I was talking about the first occurrence here: `Als externte Kraft Fext soll zunächst`

Comment (PhiSpel) [reply content/07_num_ode/03_modellierung_einer_highline_dynamsich.ipynb]: Oh yes, I see, oups, thanks!

Comment (PhiSpel) [reply content/07_num_ode/03_modellierung_einer_highline_dynamsich.ipynb]: solved in 49cf58f

File: content/07_num_ode/03_modellierung_einer_highline_dynamsich.ipynb (hunk 4 above)
Comment (PhiSpel) [reply content/07_num_ode/03_modellierung_einer_highline_dynamsich.ipynb]: solved in 49cf58f

File: content/07_num_ode/wind.m (hunk 5 above)
Comment (PhiSpel) [reply content/07_num_ode/wind.m]: solved in 49cf58f

Thank you for your comments!

**Comment by PhiSpel** [comment]
> I didn't check if the code works, but I think you checked it with your solution.
> We should open the mentioned issue before merging, so we do not forget it afterwards.

Thank you for your Input! I created #128 to follow up with a GIF in sprint 3

**Review by sandro-elsweijer — APPROVED** [review]

**Pull request merged**